File: src/main/java/com/example/dto/request/TransformerRequest.java
package com.example.dto.request;

public record TransformerRequest(
        String manufacturer,
        String modelType,
        Double ratedPowerKVA,
        Integer primaryVoltageKV,
        Integer secondaryVoltageKV,
        Double frequencyHz,
        Boolean transformerCondition,
        Boolean remoteMonitoring
) {}

File: app/src/main/java/com/example/transformmonitorapp/domain/dto/request/TransformerRequest.kt
package com.example.transformmonitorapp.domain.dto.request

data class TransformerRequest(
    val manufacturer: String,
    val modelType: String,
    val ratedPowerKVA: Double,
    val primaryVoltageKV: Int,
    val secondaryVoltageKV: Int,
    val frequencyHz: Double,
    val transformerCondition: Boolean? = null,
    val remoteMonitoring: Boolean? = null
)